specification/patient-demographics-service-api.yaml

```diff
@@ -727,6 +731,30 @@ components:
         type: string
         format: '^[0-9]+$'
         example: '555021935107'
+    BearerAuthorization:
+      in: header
+      name: Authorization
+      description: |
+        An OAuthV2 access token presented in Bearer format.
+
+        Note: This parameter is required unless interacting with the Sandbox.
+      required: false
+      schema:
+        type: string
+        format: '^Bearer\:\ [[:ascii:]]+$'
+        example: 'Bearer: g1112R_ccQ1Ebbb4gtHBP1aaaNM'
```

Comment:
Is this an apigee thing? There shouldn't be a `:` after `Bearer`

https://tools.ietf.org/html/rfc6750

Reply:
Nope that's just plain wrong on my part! Good catch